### ISSUE TITLE
Some documentation - please

### ISSUE THREAD

**Issue by heinerolli** [Xnyle/bluosec-apk#4]
@Xnyle: This is a really great App - especially for troubleshooting and optimizing settings of the OSEC Parameters for individual motors. 
I know that there is already some documentation for the App available available: [i.e. here](https://github.com/stancecoke/BMSBattery_S_controllers_firmware/wiki/05-The-BluOSEC-Android-App)

But - many items in the App are not self-explanatory (at least not to me...). Therefor: 
Could you please post somewhere some more explanations - i.e:
- What is the meaning of the different 'State Symbols'?
- What is the meaning of the different wave tables? (Mid Clamp, Nip&Tuck, ....?)
- When I look at the Phase B current diagram: I can make the curve flatter by slight adjustments of the 6 Hall-Angle values (in std. riding mode, not in 'DC static Zero'). Is this the right approach to fine-tune the angles?
- Are there other possibilities to use the App for fine-tuning? (i.e. the overall 'Correction angle'?)
- ....

Many people would benefit from that - and this App could get a lot more users.

**Comment by Xnyle** [comment]
Yeah a bit of a shame that there isn't (also further developement) but I currently don't have the time to work on the app let alone write some documentation. I mainly created it to suit my needs and that it does :-)

**Comment by heinerolli** [comment]
Thanks for the feedback - I can completely understand (lack of time aso)!

Would it be possible though that you feedback briefly in what scenario you are using the App, where it works reliably? 
I.e. which branch of https://github.com/stancecoke/BMSBattery_S_controllers_firmware are you using and what features of that branch are you using / are working? 
(My use case: I want to use the simple Throttle input for a project, but I would really like to incorpaorate your App to use 'Assist level affects Throttle" with it. And that does not work with the master branch - the controller crashes / hangs very often)

**Comment by cnheider** [comment]
> but I currently don't have the time to work on the app let alone write some documentation.

Let us help you out then :)
